app/views/accounts/_accomplished_objective.html.slim

```diff
@@ -12,11 +12,6 @@ tr.for-desktop
     .has-text-centered
       p.pt-2.is-black-budgeting-kid-color
         | #{ accomplished_objective.accomplished_at.strftime('%b %d, %y') }
-  td
```

Comment:
Do I understand correctly that it removed the delete goal button completely? My point was we should hide it only for accomplished goals.

Reply:
looks like it's working correctly.